src/objects.rs

```diff
@@ -17,7 +17,7 @@ pub struct MorayObject {
     pub _etag: String,
     pub _id: u64,
     pub _mtime: u64,
-    // TODO: _txn_snap:
+    pub _txn_snap: Option<u64>,
```

Comment:
This was just added for completeness.  I left it out originally because I didn't know its value or type.
https://github.com/joyent/moray/blob/master/docs/index.md#getobject